server/trucks.js

```diff
@@ -248,9 +248,21 @@ var foodTrucks = [
 	}
 ];
 
+
+//look through the schedule for each truck
+// if the schedule contains the name of the day in getDayName
+// include the Truck in the availableTrucks
+function filterByDay(dayName) {
+	 _.filter(foodTrucks, function (truck) {
+          _.any(truck.schedule === dayName);
```

Comment:
Using _.any is a good idea here, but it doesn't function like a conditional check. Instead it takes two arguments: an array, and a function to call that will check each value in the list until it finds a match, then it will return that value. Then, _.filter needs to return what _.any returned: `function filterByDay(dayName) { return _.filter(foodTrucks, function (truck) { return _.any(truck.schedule, function (day) { return dayName === day.name }); }); }